src/renderer/utils/ModData.ts

```diff
@@ -0,0 +1,37 @@
+import { Mod, ModTrack, RollingReleaseModel } from "renderer/utils/InstallerConfiguration";
+import { GitVersions } from "@flybywiresim/api-client";
+
+export type ReleaseInfo = {
+    name: string,
+    releaseDate: Date,
+    changelogUrl?: string,
+}
+
+export class ModData {
+
+    static async latestVersionForTrack(mod: Mod, track: ModTrack): Promise<ReleaseInfo> {
+        if (track.releaseModel.type === 'release') {
+            return this.latestVersionForReleasedTrack(mod);
+        } else if (track.releaseModel.type === 'rolling') {
+            return this.latestVersionForRollingTrack(mod, track.releaseModel);
+        }
```

Comment:
These types should be more specific. Something like `github/release` and `github/branch` so we can have other types which don't fetch the info from GitHub.

Reply:
Isn't that a bit of premature abstraction

Reply:
Better to just rename it now. If we need to do it later we'll have to think about backwards compatibility if the addon data gets fetched from an API or CDN.